topics/sentinel.md

```diff
@@ -583,26 +583,36 @@ so forth.
+* **CLIENT** This command manages client connections. For more information refer to the its subcommands' pages.
+* **COMMAND** This command returns information about commands. For more information refer to the `COMMAND` command and its various subcommands.
+* **HELLO** Switch the connection's protocol. For more information refer to the `HELLO` command.
+* **INFO** Return information and statistics about the Sentinel server. For more information see the `INFO` command.
 * **PING** This command simply returns PONG.
-* **SENTINEL masters** Show a list of monitored masters and their state.
-* **SENTINEL master `<master name>`** Show the state and info of the specified master.
-* **SENTINEL replicas `<master name>`** Show a list of replicas for this master, and their state.
-* **SENTINEL sentinels `<master name>`** Show a list of sentinel instances for this master, and their state.
-* **SENTINEL get-master-addr-by-name `<master name>`** Return the ip and port number of the master with that name. If a failover is in progress or terminated successfully for this master it returns the address and port of the promoted replica.
-* **SENTINEL reset `<pattern>`** This command will reset all the masters with matching name. The pattern argument is a glob-style pattern. The reset process clears any previous state in a master (including a failover in progress), and removes every replica and sentinel already discovered and associated with the master.
-* **SENTINEL failover `<master name>`** Force a failover as if the master was not reachable, and without asking for agreement to other Sentinels (however a new version of the configuration will be published so that the other Sentinels will update their configurations).
-* **SENTINEL ckquorum `<master name>`** Check if the current Sentinel configuration is able to reach the quorum needed to failover a master, and the majority needed to authorize the failover. This command should be used in monitoring systems to check if a Sentinel deployment is ok.
-* **SENTINEL flushconfig** Force Sentinel to rewrite its configuration on disk, including the current Sentinel state. Normally Sentinel rewrites the configuration every time something changes in its state (in the context of the subset of the state which is persisted on disk across restart). However sometimes it is possible that the configuration file is lost because of operation errors, disk failures, package upgrade scripts or configuration managers. In those cases a way to to force Sentinel to rewrite the configuration file is handy. This command works even if the previous configuration file is completely missing.
+* **ROLE** This command returns the string "sentinel" and a list of monitored masters. For more information refer to the `ROLE` command.
+* **SHUTDOWN** Shut down the Sentinel instance.
```

Comment:
was a little bit hard to review.
for some reason github shows that you changed all the lines about the SENTINEL commands.
but comparing them one by one i see they are unchanged, and all you did is add the section at the top with the generic management APIs.

I see you also added COMMAND which is not yet merged, but we can assume is gonna be soon.
i suppose we need a versioning paragraph here to mention that ACL and COMMAND are new to 6.2.

also, i see all the PUBSUB commands are missing from this list. they are documented below, but maybe this list should have some note and a link to the other section.

Reply:
I uppercased the subcommands in these lines, perhaps that's why.

WRT COMMAND - exactly.

Pub/Sub - as it is covered in another section I didn't put it in, but I also felt it was missing in terms of completeness. Will add.